### PR TITLE
clean up table 8

### PR DESCRIPTION
**Pull request by AreezG**
table 8 did not follow the conventions we've established, so fix it

### CODE REVIEW

File: table8.do [hunk 1]
```diff
@@ -1,174 +1,168 @@
...
-	
-		generate id = _n
-		sort temp_city order
+/* Stata Do File for Table 8 */
+/* Written by: Matthew D. Webb */
+/* Updated: July 30th, 2024, by Sunny Karim */
+
+clear
+
+import delimited "${Data}/Table 8.csv", case(preserve)
```
Comment (shichen14) [inline table8.do:7]: This csv file is not available from the 'Data' folder. Suggest to remove this line and let line 19 call the dta file instead.

File: table8.do [hunk 2]
```diff
@@ -1,174 +1,168 @@
...
+
+save "${OUTPUT}/Table 8.dta", replace
```
Comment (shichen14) [inline table8.do:9]: See line 7 comment. Suggest to remove this line.

File: table8.do [hunk 3]
```diff
@@ -1,174 +1,168 @@
...
+	
+	
+	rename ofcolor oc
+	drop race
+	rename apracex race
+	
+tostring zip_ad, replace
+
+// clean city names
+do "${CODE}/temp2_data_cleaner.do"
```
Comment (shichen14) [inline table8.do:50]: This "temp2" version of cleaner do.file cannot be found in the code directory.

File: table8.do [hunk 4]
```diff
@@ -1,174 +1,168 @@
...
+
+
+save "${OUTPUT}/Table8_adjustedcities_score.dta", replace
```
Comment (shichen14) [inline table8.do:53]: See line 50 comment. Need "temp2" version of cleaner do.file to proceed.

**Review by shichen14 — COMMENTED** [review]
The main file is unable to complete executing due to issues in Table 8 do.file. Please see comments in the Table 8 do.file for details.

**Comment by AreezG** [comment]
looks like @SunnyRKarim cleaned up this file as well, closing

**Pull request closed without merging**